src/helpers/parser/listItemParser.js

```diff
@@ -274,7 +274,18 @@ const parseSUERequests = (data) => {
       status: item?.status,
       title: item?.title,
       updatedDate: item?.updated_datetime,
-      zipcode: item?.zipcode
+      zipcode: item?.zipcode,
+      routeName: ScreenName.Detail,
+      params: {
+        title: texts.detailTitles.pointOfInterest,
+        query: QUERY_TYPES.SUE.REQUESTS_WITH_SERVICE_REQUEST_ID,
+        queryVariables: { id: item.service_request_id },
+        rootRouteName: ROOT_ROUTE_NAMES.SUE,
+        details: {
+          ...item,
+          title: item.title
```

Comment:
if you destruct item, title is already item.title.
`details: item` should be enough here

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/960/commits/754a44edc67676d05ca82e810e413c9d9a8c8e77